contentcuration/contentcuration/frontend/shared/data/resources.js

```diff
@@ -1,20 +1,28 @@
 import Dexie from 'dexie';
 import findIndex from 'lodash/findIndex';
 import flatMap from 'lodash/flatMap';
+import isArray from 'lodash/isArray';
+import isFunction from 'lodash/isFunction';
 import matches from 'lodash/matches';
 import overEvery from 'lodash/overEvery';
+import uniq from 'lodash/uniq';
+
 import uuidv4 from 'uuid/v4';
 import channel from './broadcastChannel';
+import { getChangeSet } from './changes';
 import {
   CHANGE_TYPES,
-  FETCH_SOURCE,
+  CHANGES_TABLE,
+  IGNORED_SOURCE,
```

Comment:
What does this mean?

Reply:
Every Dexie transaction can have a `source` set on it. Transactions that are loading from the server used `FETCH_SOURCE` which was then ignored when capturing changes that need syncing. Of course, if we fetched the data, we don't need to sync that back. I simply renamed the constant since I was expanding the *ignore* usage.

Reply:
Ah gotcha! Do you mind adding a comment for future clarification?